ui/ui-components/components/record/AddSampleRecordForm.tsx

```diff
@@ -27,6 +28,8 @@ const AddSampleRecordForm: React.FC<Props> = ({
   onSubmitComplete,
   properties,
 }) => {
+  // TODO: Remove me when 180627104-better-add-sample-record-logic-for-invalid-ids is merged
+  modelId = useGrouparooModel().model.id;
```

Comment:
Can you explain the issue behind this?

Reply:
modelId is no longer passed in from the server-side props. This was a stopgap for testing that I fixed in my other PR.
https://github.com/grouparoo/grouparoo/pull/2929/files#diff-d270cd4e98210e75e8d1c9eecb16c95913f4f08cf6ff0c19a9366f42971e708cR19